src/convertToObject.js

```diff
@@ -1,18 +1,35 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
- *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cssClass = {};
+  const sourceArr = sourceString.split('');
+
+  let sourseArr = sourceArr.filter((item, i) => {
+    if (item === '\n' || (item === ' ' && sourceArr[i - 1] === ' ')) {
+      return false;
+    }
+
+    return true;
+  });
+
+  sourseArr = sourseArr.join('');
+  sourseArr = sourseArr.split(';');
```

Comment:
we can just split it by ';'

Reply:
and then remove spaces using the `trim` method

Reply:
thanks, I went step by step, through the debugger and the console, and confused myself